### ISSUE TITLE
loss=3.x

### ISSUE THREAD

**Issue by ShifengJin** [shekkizh/FCN.tensorflow#106]
Hi ,
    the loss=3.x  when i train. where can i find your model?
